test/TEST-16-ISSUE-518/test.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/python
+
+#
+#  Copyright 2017 Michal Sekletar <msekleta@redhat.com>
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+#
+#  systemd is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public License
+#  along with python-systemd; If not, see <http://www.gnu.org/licenses/>.
+
+import unittest
+import time
+import os
+import tempfile
+
+from pydbus import SystemBus
```

Comment:
or perhaps use `gi.repository import Gio, GLib` directly, i. e. the [GDBus API](https://developer.gnome.org/gio/stable/gdbus-convenience.html): like `Gio.bus_get_sync(Gio.BusType.SYSTEM,) None)` or `Gio.DBusProxy.new_sync()` etc. pydbus just wraps that, and at least for these simple cases using a wrapper doesn't really buy much.